packages/app/src/cli/models/extensions/specifications/flow_trigger.ts

```diff
@@ -15,6 +17,7 @@ const FlowTriggerSettingsSchema = FieldSchema.extend({
 
 export const FlowTriggerExtensionSchema = BaseSchemaWithHandle.extend({
   type: zod.literal('flow_trigger'),
+  schema: zod.string().optional(),
```

Comment:
@omnisyle it looks like this PR is going with option 1.
https://github.com/Shopify/flow/pull/19921#discussion_r1281077203